modules/sdk/ant-bnd/src/test/java/com/liferay/ant/bnd/jsp/JspAnalyzerPluginTest.java

```diff
@@ -180,6 +180,58 @@ public void testRemoveDuplicateTaglibRequirements() throws Exception {
 		Assert.assertEquals(requireCapability1, requireCapability2);
 	}
 
+	@Test
+	public void testTaglibRequirementsWithJakartaJstlCoreUri()
```

Comment:
I don't think `Jstl` applies to any of the core Jakarta URIs. Can you please double-check that?